xili-tidy-tags/xili-tidy-tags.php

```diff
@@ -498,14 +485,24 @@ function xili_get_object_terms( $object_ids, $taxonomies, $args = array() ) {
 	$args = array_merge ( $defaults, $args );
 	extract ($args);
 
+	settype( $taxonomies, 'array' );
+	foreach ( $taxonomies as $taxonomy ) {
+		if ( ! is_taxonomy($taxonomy) )
+			return new WP_Error('invalid_taxonomy', __('Invalid Taxonomy'));
+	}
 
-	if (!is_array($sub_groups)) $sub_groups = array($sub_groups);
+	settype( $object_ids, 'array' );
```

Comment:
These two lines are functionally equivalent, but I think `settype` is a lot cleaner and shorter.